ludicrousdb/includes/class-ludicrousdb.php

```diff
@@ -1615,7 +1615,8 @@ public function has_cap( $db_cap, $dbh_or_table = false ) {
 						return version_compare( $client_version, '5.5.3', '>=' );
 					}
 				}
-			case 'utf8mb4_520' : // @since 4.6.0
+				break;
+			case 'utf8mb4_520': // @since 4.6.0
```

Comment:
Since when are spaces here not preferred? That blows my mind. There have always been spaces after switches.